src/Client.php

```diff
@@ -155,7 +155,7 @@ public function __debugInfo()
      * @param string $databaseName Name of the database to select
      * @return Database
      */
-    public function __get($databaseName)
+    public function __get(string $databaseName)
```

Comment:
Noted that the [magic methods use a type hint](https://www.php.net/manual/en/language.oop5.overloading.php#object.get). I assume this might be problematic if they did not, unless it'd be permitted by type narrowing anyway.

Reply:
That is correct - except that for method parameters type narrowing is not allowed, only widening ;)